hugegraph-llm/api/src/text2kg/text_to_kg.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+
+from api.src.llm.openai import OpenAIChat
+from text2kg.data_to_data import DataDisambiguation
+from text2kg.data_to_kg import DataToKg
+from text2kg.text_to_data import TextToData
+
+
+class TextToKg:
+    def __init__(self, name):
+        self.name = name
+        self.text2kg = []
+
+    def text2data(self, llm):
+        self.text2kg.append(TextToData(llm=llm))
+        return self
+
+    def data2data(self, llm):
```

Comment:
wan we provide another name to distinguish the second data?
like method names:

1. parse_text_to_data
2. disambiguate_data
3. commit_data_to_kg